tests/test_gitlab/events/pull_request.py

```diff
@@ -0,0 +1,361 @@
+# SPDX-FileCopyrightText: 2024-2025 Contributors to the Fedora Project
+#
+# SPDX-License-Identifier: LGPL-3.0-or-later
+
+"""
+Event specification for GitLab "issues" event
+"""
+
+headers = {
+    "host": "w2fm.gridhead.net",
+    "user-agent": "GitLab/18.1.0-pre",
+    "content-length": 8172,
+    "accept": "*/*",
+    "accept-encoding": "gzip, br",
+    "connection": "keep-alive",
+    "content-type": "application/json",
+    "x-gitlab-event": "Merge Request Hook",
+    "x-gitlab-webhook-uuid": "9cf056fa-f86c-49bb-93f0-b9e705eee5df",
+    "x-gitlab-event-uuid": "ea4bf54d-f330-4702-94e0-dda5ee03a57f",
+    "x-forwarded-for": "34.74.226.47",
+    "x-forwarded-proto": "https",
+}
+
+body = {
+    "object_kind": "merge_request",
+    "event_type": "merge_request",
+    "user": {
+        "id": 16997011,
+        "name": "Shounak Dey",
+        "username": "sdglitched",
+        "avatar_url": "https://secure.gravatar.com/avatar/9458b3a00e2f3f262b05309bd9a43a233c28689e6cbae811b198cd5ee9fb3653?s=80&d=identicon",
+        "email": "[REDACTED]",
+    },
+    "project": {
+        "id": 69912300,
+        "name": "w2fm-test",
+        "description": None,
+        "web_url": "https://gitlab.com/sdglitched/w2fm-test",
+        "avatar_url": None,
+        "git_ssh_url": "git@gitlab.com:sdglitched/w2fm-test.git",
+        "git_http_url": "https://gitlab.com/sdglitched/w2fm-test.git",
+        "namespace": "Shounak Dey",
+        "visibility_level": 0,
+        "path_with_namespace": "sdglitched/w2fm-test",
+        "default_branch": "main",
+        "ci_config_path": "",
+        "homepage": "https://gitlab.com/sdglitched/w2fm-test",
+        "url": "git@gitlab.com:sdglitched/w2fm-test.git",
+        "ssh_url": "git@gitlab.com:sdglitched/w2fm-test.git",
+        "http_url": "https://gitlab.com/sdglitched/w2fm-test.git",
+    },
+    "object_attributes": {
+        "assignee_id": 16997011,
+        "author_id": 16997011,
+        "created_at": "2025-05-16 16:04:39 UTC",
+        "description": "Testing merge request\n\nSigned-off-by: Shounak Dey <shounakdey@ymail.com>",
```

Comment:
Please remove your email address and add some random placeholder here like `testaddr@testaddr.test`.